docs/reference/method/MongoDBCollection-watch.txt

```diff
@@ -0,0 +1,50 @@
+================================
+MongoDB\\Collection::watch()
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Collection::watch()
+
+   Executes a :manual:`change stream </changeStreams>` operation on the
```

Comment:
This will render as https://docs.mongodb.com/manual/changeStreams, which is currently a 404 but should point to the content at https://docs.mongodb.com/master/changeStreams/ once 3.6 is the default version for the server docs. Feel free to confirm with @kay-kim if you haven't already. Otherwise, this LGTM.